ch3/01.py

```diff
@@ -0,0 +1,8 @@
+import random
```

Comment:
start with #!/usr/bin/env, see ex0.py

also add the exercise text in the comment